web/src/beta/lib/core/engines/Cesium/Feature/Model/index.tsx

```diff
@@ -94,6 +122,72 @@ export default function Model({ id, isVisible, property, geometry, layer, featur
     [property?.near, property?.far],
   );
 
+  const imageBasedLighting = useMemo(() => {
+    if (
+      !property?.specularEnvironmentMaps &&
+      !property?.sphericalHarmonicCoefficients &&
+      !sceneProperty?.light?.specularEnvironmentMaps &&
+      !sceneProperty?.light?.sphericalHarmonicCoefficients
+    )
+      return;
+
+    const ibl = new ImageBasedLighting();
+    const specularEnvironmentMaps = property?.specularEnvironmentMaps
+      ? property?.specularEnvironmentMaps
+      : sceneProperty?.light?.specularEnvironmentMaps;
+    const sphericalHarmonicCoefficients = property?.sphericalHarmonicCoefficients
+      ? property?.sphericalHarmonicCoefficients
+      : sceneProperty?.light?.sphericalHarmonicCoefficients;
+    const imageBasedLightIntensity =
+      property?.imageBasedLightIntensity ?? sceneProperty?.light?.imageBasedLightIntensity;
```

Comment:
```suggestion
    const specularEnvironmentMaps =
      property?.specularEnvironmentMaps ?? sceneProperty?.light?.specularEnvironmentMaps;
    const sphericalHarmonicCoefficients =
      property?.sphericalHarmonicCoefficients ?? sceneProperty?.light?.sphericalHarmonicCoefficients;
    const imageBasedLightIntensity =
      property?.imageBasedLightIntensity ?? sceneProperty?.light?.imageBasedLightIntensity;
```
Maybe we can align the expression of the code?

Reply:
Updated. 🙏🏻